spring-integration-core/src/test/java/org/springframework/integration/router/RecipientListRouterTests.java

```diff
@@ -346,17 +353,21 @@ public void applySequenceEnabled() {
 				.isEqualTo(message.getHeaders().getId());
 	}
 
-	@Test(expected = IllegalArgumentException.class)
+	@Test
 	public void nullChannelListRejected() {
 		RecipientListRouter router = new RecipientListRouter();
-		router.setChannels(null);
+		assertThatThrownBy(() -> {
+			router.setChannels(null);
+		}).isInstanceOf(IllegalArgumentException.class);
```

Comment:
Doesn't look like we need body for the lambda. I mean `{}` and `;` in the end of sentence are redundant.